mmcv/ops/csrc/parrots/modulated_deform_conv_pytorch.h

```diff
@@ -2,7 +2,6 @@
 #ifndef MODULATED_DEFORM_CONV_PYTORCH_H
 #define MODULATED_DEFORM_CONV_PYTORCH_H
 #include <torch/extension.h>
-using namespace at;
```

Comment:
Why was the line removed?

Reply:
Sorry about missing when recovering from `types.h` to from `extension.h`. The compiling process keeps successful after adding this line.